app/components/common/Navigation.vue

```diff
@@ -0,0 +1,534 @@
+<script>
+  const CODECOMBAT = 'codecombat'
+  const CODECOMBAT_CHINA = 'koudashijie'
+  const OZARIA = 'ozaria'
+  const OZARIA_CHINA = 'aojiarui'
+
+  // Use in local development to inform which repo you are in
+  const LOCALHOST_FAKE_HOST = () => application.isProduction() ? null : CODECOMBAT // 'codecombat' | 'ozaria'
+
+  /**
+   * Unified navigation bar component between CodeCombat and Ozaria.
+   * This must be copied exactly to the Ozaria repo.
+   */
+  export default Vue.extend({
+    computed: {
+      isOldBrowser () {
```

Comment:
Can we just move this function to `app/core/utils.coffee` and call it from there, rather than have a duplicate implementation both here and in RootView.coffee?

Reply:
I can do that!